### ISSUE TITLE
Window stacks miss mouse inputs near the bottom of the frame

### ISSUE THREAD

**Issue by zeptofine** [pop-os/cosmic-epoch#1251]
In some apps, there is an unusable area at the bottom of the window when it is stacked. In this area, the window does not consider mouse input. This makes certain apps difficult to use because their buttons are smaller/almost as small as the margin of missing area, such as some QT popups, and VSCode (and derivatives).
It doesn't seem to matter if there are any other windows in the stack, the window just needs to be stacked. It does not matter if it is focused or not.

The purple line is my configured window outline, The red line indicates the approximate path my mouse takes, and the green line indicates where I feel the cutoff area is.

Not stacked

![screenshot-2024-11-15-22-10-57-annotated](https://github.com/user-attachments/assets/56bdeaa1-e087-471f-b7b7-c7e44bc16437)

Stacked

![screenshot-2024-11-15-22-06-32-annotated](https://github.com/user-attachments/assets/cde73e24-4178-4530-9029-1251a240c2bc)

**Comment by git-f0x** [comment]
Can reproduce. It seems to be a similar issue as pop-os/cosmic-comp#951.

**Comment by zeptofine** [comment]
Extra things I've noticed an hour later:
The height of the area seems roughly equivalent to the height of the stack bar at the top, so perhaps whatever is calculating the "bounding box" of inputtable space is resizing too much / too little based on the bar's size

I can focus on another window and click into this space on a stacked window, and the _focus_ changes but the _click / movement_ doesn't register

**Comment by git-f0x** [comment]
Yeah, it's the exact same behavior as the mentioned issue (the area was the height of SSDs there).

**Comment by git-f0x** [comment]
This has been fixed.
